src/AvaTaxApi.cs

```diff
@@ -3394,9 +3394,9 @@ public FileResult BuildPointOfSaleDataFile(PointOfSaleDataRequestModel model)
         public String ChangePassword(PasswordChangeModel model)
         {
             var path = new AvaTaxPath("/api/v2/passwords");
-            return RestCallString("Put", path, model);
+            return ExecuteRestCall("Put", path, model, BodyAsString);
```

Comment:
It might be simpler to just modify RestCallString to do logging on its own, wouldn't it?